### PR TITLE
Some splits for better modding options

### PR DESCRIPTION
**Pull request by onemantooo**
Hello.
Recently i've tried to add some options to spawned patrols, and one caused a problem - i wanted to add an options to spawn only male or only female AIs via my mod configuration. But to accomplish that - i need to fully override or copy `SpawnAI` method which causes obvious problem - it can break Expansion functionality after some of your updates. 

So i took all my bravery to send you this PR. It's just splits `SpawnAI` to few separated methods.

If you won't accept this one, maybe i can send you a PR with male/female functionality?

### CODE REVIEW

File: DayZExpansion/AI/Scripts/4_World/DayZExpansion_AI/Classes/Patrols/eAIDynamicPatrol.c [hunk 1]
```diff
@@ -133,8 +133,18 @@ class eAIDynamicPatrol : eAIPatrol
...
+
+		return ai;
+	}
+
+	private Object CreateAIEntity(vector pos) {
+		return GetGame().CreateObject(GetRandomAI(), pos);
+	}
+
+	private void ConfigureAi(eAIBase ai, vector pos) {
 		ai.SetPosition(pos);
```
Comment (onemantooo) [inline DayZExpansion/AI/Scripts/4_World/DayZExpansion_AI/Classes/Patrols/eAIDynamicPatrol.c:148]: Not sure why is this needed cos we already passed position to `CreateObject` but kept it untouched.

Comment (salutesh) [reply DayZExpansion/AI/Scripts/4_World/DayZExpansion_AI/Classes/Patrols/eAIDynamicPatrol.c:148]: Options to set the specific AI class names used to spawn the AI patrol units have been added in the last update:
https://github.com/salutesh/DayZ-Expansion-Scripts/blob/263948278268d5cda105caa0d9145c23fc38a480/DayZExpansion/AI/Scripts/4_World/DayZExpansion_AI/Classes/Patrols/eAIDynamicPatrol.c#L141

**Comment by LieutenantMaster** [comment]
In the latest update now, thank you for the suggestion <3
